pages/linux/runcon.md

```diff
@@ -0,0 +1,20 @@
+# runcon
+
+> Run a program in a different SELinux security context.
+> With neither CONTEXT nor COMMAND, print the current security context.
+
+- Determine the current domain:
+
+`runcon`
+
+- Specify the domain to run a command in:
+
+`runcon -t CONTEXT-TYPE_t {{command}}`
```

Comment:
This has a - in between CONTEXT and TYPE, but the later examples use _. Which is it ?

Also, don't these need to be inside tokens (`{{}}`)